src/main/drivers/max7456.c

```diff
@@ -475,9 +471,12 @@ void max7456Brightness(uint8_t black, uint8_t white)
 
     if (reg != previousBlackWhiteRegister) {
         previousBlackWhiteRegister = reg;
-        for (int i = MAX7456ADD_RB0; i <= MAX7456ADD_RB15; i++) {
-            spiWriteReg(dev, i, reg);
+        STATIC_DMA_DATA_AUTO uint8_t buf[32];
+        for (int i = MAX7456ADD_RB0, j = 0; i <= MAX7456ADD_RB15; i++) {
+            buf[j++] = i;
+            buf[j++] = reg;
         }
+        spiReadWriteBuf(dev, buf, NULL, sizeof (buf));
```

Comment:
There should be no space after ```sizeof```

Reply:
784 instances with a space and 3,331 without across the source so there is an argument to change. There are three instances with the space in this PR which is what I've used for decades. Note that the [C99 Standard](http://www.open-std.org/jtc1/sc22/WG14/www/docs/n1256.pdf) shows the use of a space. Any further thoughts?

Reply:
Just found that our coding standard, https://github.com/betaflight/betaflight/blob/master/docs/development/CodingStyle.md, explicitly states no spaces after sizeof.

Reply:
Fixed

Reply:
Personally I like to have the space here as it indicates it's an compile time operator and not a runtime function.

Reply:
I have no preference here. Only requesting the change because of the betaflight codingstyle